api/graphql/mutations.js

```diff
@@ -10,9 +10,20 @@ import {
   validateThreadData
 } from '../models/post/util'
 
+function convertGraphqlUserSettingsData (data) {
+  return transform(data, (result, value, key) => {
```

Comment:
just fyi, if you're not going to remove/add/merge keys, `mapValues` does the job too